app/javascript/components/users-dropdown.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <clickable-dropdown
+      :body-title="dropdownTitle"
+      :no-items-message="noUsersMessage"
+      :items="users"
+      @item-clicked="onItemClicked"
+    /><br>
+    <div v-for="(user, index) in selected"
+      :key="user.id"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 9 spaces but found 6 spaces                               vue/html-indent

Reply:
No entiendo por qué este me dice que se espera una indentación de 9 espacios y más adelante en exactamente la misma línea del archivo me dice que se esperan 6